subsys/bluetooth/host/smp.c

```diff
@@ -204,6 +205,9 @@ struct bt_smp {
 
 	/* Delayed work for timeout handling */
 	struct k_work_delayable		work;
+
+	/* A work item for persisting ccc and cf data in the system workqueue */
+	struct k_work			work_settings_ccc_and_cf_save;
```

Comment:
If you're really not going to use the existing `identity_resolved` callback (which I still hope you'd reconsider for this PR), then at least please keep the changes to `smp.c` as minimal as possible. That means putting any additional work structs into `gatt.c` and not `smp.c`.

The name of the variable is also unnecessarily long (you really seem to be a fan of long symbol names 😉 ).

Reply:
Okidoki :-) Yes I am ;-) But I will find something shorter.

Reply:
I am now using the identity_resolved callback. I agree this is nicer.

Reply:
This one no longer used.